test/index.js

```diff
@@ -19,6 +19,7 @@ describe('Klaviyo', function () {
       listId: 'baVTu8',
       sendAnonymous: true,
       enforceEmail: false,
+      useSegmentSpec: false
```

Comment:
because that is the current behavior. server side doesn't send `Order Completed` but `Placed Order`